<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_bt_dish_kitchen_dish_state</class>
 <widget class="QDialog" name="w_bt_dish_kitchen_dish_state">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color:#ff546b;</string>
     </property>
     <property name="text">
      <string>显示当天的数据</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="lds_tableView" name="tableView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton_lost">
           <property name="text">
            <string>漏打单</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_printed">
           <property name="text">
            <string>已打单</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_lost_clear">
           <property name="text">
            <string>清除漏打单</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="lds_toolbar_button" name="pushButton_up">
         <property name="text">
          <string>上一页</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_toolbar_button" name="pushButton_down">
         <property name="text">
          <string>下一页</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_toolbar_button" name="pushButton_refresh">
         <property name="text">
          <string>刷新</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="lds_toolbar_button" name="pushButton_kitchen_print">
         <property name="text">
          <string>厨打打印机</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_toolbar_button" name="pushButton_exit">
         <property name="text">
          <string>退出</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_tableView</class>
   <extends>QTableView</extends>
   <header>lds_tableview.h</header>
  </customwidget>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
